### PR TITLE
Work on capacity issues / Strategy QA

### PR DESCRIPTION
**Pull request by Camwyn**
Tickets: https://central.tri.be/issues/80625 and https://central.tri.be/issues/86035

Required: https://github.com/moderntribe/event-tickets-plus/pull/367

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $post_id is never used | [Tickets_Handler.php:211](https://github.com/moderntribe/event-tickets/blob/fd85707bc3d7ffb16bdc46059f26958400df0a5f/src/Tribe/Tickets_Handler.php#L211)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $post_id is never used | [Tickets_Handler.php:210](https://github.com/moderntribe/event-tickets/blob/6b225871d08ca3a9622dca904fe52ab9f5b0e5c9/src/Tribe/Tickets_Handler.php#L210)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $post_id is never used | [Tickets_Handler.php:210](https://github.com/moderntribe/event-tickets/blob/f6efa2630e84d285ab42555d4d63ed086f515c7a/src/Tribe/Tickets_Handler.php#L210)

> via jenkins/codesniffer-phpcs

File: src/admin-views/edit_admin_panel.php [hunk 1]
```diff
@@ -192,8 +195,8 @@ class="tribe-timepicker tribe-field-end_time ticket_field"
 			?>
 			<div class="ticket_bottom">
 					<input type="hidden" name="ticket_id" id="ticket_id" class="ticket_field" />
-					<input type="button" id="ticket_form_save" name="ticket_form_save" value="<?php esc_attr_e( 'Save ticket', 'event-tickets' ); ?>" class="button-primary" />
-					<input type="button" id="rsvp_form_save" name="ticket_form_save" value="<?php esc_attr_e( 'Save RSVP', 'event-tickets' ); ?>" class="button-primary" />
+					<input type="button" id="ticket_form_save" name="ticket_form_save" value="<?php esc_attr_e( 'Save ticket', 'event-tickets' ); ?>" class="button-primary tribe-dependent" data-depends="#Tribe__Tickets__RSVP_radio" data-condition-is-not-checked />
```
Comment (bordoni) [inline src/admin-views/edit_admin_panel.php]: Multiple Lines for Dependent.

File: src/admin-views/edit_admin_panel.php [hunk 2]
```diff
@@ -23,18 +23,21 @@
...
-			<?php // @TODO: Do these need to get renamed for RSVPs? ?>
-			<h4 id="ticket_title_add" class="ticket_form_title"><?php esc_html_e( 'Add new ticket', 'event-tickets' ); ?></h4>
-			<h4 id="ticket_title_edit" class="ticket_form_title"><?php esc_html_e( 'Edit ticket', 'event-tickets' ); ?></h4>
-			<h4 id="rsvp_title_add" class="ticket_form_title"><?php esc_html_e( 'Add new RSVP', 'event-tickets' ); ?></h4>
-			<h4 id="rsvp_title_edit" class="ticket_form_title"><?php esc_html_e( 'Edit RSVP', 'event-tickets' ); ?></h4>
+			<div class="tribe-dependent"  data-depends="#Tribe__Tickets__RSVP_radio" data-condition-is-not-checked>
+				<h4 id="ticket_title_add" class="ticket_form_title tribe-dependent" data-depends="#ticket_id" data-condition-is-empty><?php esc_html_e( 'Add new ticket', 'event-tickets' ); ?></h4>
+				<h4 id="ticket_title_edit" class="ticket_form_title tribe-dependent" data-depends="#ticket_id" data-condition-is-not-empty><?php esc_html_e( 'Edit ticket', 'event-tickets' ); ?></h4>
+			</div>
+			<div class="tribe-dependent"  data-depends="#Tribe__Tickets__RSVP_radio" data-condition-is-checked>
```
Comment (bordoni) [inline src/admin-views/edit_admin_panel.php]: Multiple Lines for Dependent.

File: src/admin-views/edit_admin_panel.php [hunk 3]
```diff
@@ -23,18 +23,21 @@
...
+				<h4 id="rsvp_title_add" class="ticket_form_title tribe-dependent" data-depends="#ticket_id" data-condition-is-empty><?php esc_html_e( 'Add new RSVP', 'event-tickets' ); ?></h4>
```
Comment (bordoni) [inline src/admin-views/edit_admin_panel.php]: Multiple Lines for Dependent.

File: src/admin-views/edit_admin_panel.php [hunk 4]
```diff
@@ -23,18 +23,21 @@
...
+				<h4 id="rsvp_title_edit" class="ticket_form_title tribe-dependent" data-depends="#ticket_id" data-condition-is-not-empty><?php esc_html_e( 'Edit RSVP', 'event-tickets' ); ?></h4>
```
Comment (bordoni) [inline src/admin-views/edit_admin_panel.php]: Multiple Lines for Dependent.

File: src/admin-views/edit_admin_panel.php [hunk 5]
```diff
@@ -23,18 +23,21 @@
 
 	<div id="ticket_form" class="ticket_form tribe_sectionheader">
 		<div id="ticket_form_table" class="eventtable ticket_form">
-			<?php // @TODO: Do these need to get renamed for RSVPs? ?>
-			<h4 id="ticket_title_add" class="ticket_form_title"><?php esc_html_e( 'Add new ticket', 'event-tickets' ); ?></h4>
-			<h4 id="ticket_title_edit" class="ticket_form_title"><?php esc_html_e( 'Edit ticket', 'event-tickets' ); ?></h4>
-			<h4 id="rsvp_title_add" class="ticket_form_title"><?php esc_html_e( 'Add new RSVP', 'event-tickets' ); ?></h4>
-			<h4 id="rsvp_title_edit" class="ticket_form_title"><?php esc_html_e( 'Edit RSVP', 'event-tickets' ); ?></h4>
+			<div class="tribe-dependent"  data-depends="#Tribe__Tickets__RSVP_radio" data-condition-is-not-checked>
```
Comment (bordoni) [inline src/admin-views/edit_admin_panel.php]: Multiple Lines for Dependent.

File: src/admin-views/edit_admin_panel.php [hunk 6]
```diff
@@ -23,18 +23,21 @@
...
+				<h4 id="ticket_title_add" class="ticket_form_title tribe-dependent" data-depends="#ticket_id" data-condition-is-empty><?php esc_html_e( 'Add new ticket', 'event-tickets' ); ?></h4>
```
Comment (bordoni) [inline src/admin-views/edit_admin_panel.php]: Multiple Lines for Dependent.

File: src/admin-views/edit_admin_panel.php [hunk 7]
```diff
@@ -23,18 +23,21 @@
...
+				<h4 id="ticket_title_edit" class="ticket_form_title tribe-dependent" data-depends="#ticket_id" data-condition-is-not-empty><?php esc_html_e( 'Edit ticket', 'event-tickets' ); ?></h4>
```
Comment (bordoni) [inline src/admin-views/edit_admin_panel.php]: Multiple Lines for Dependent.

**Pull request merged**